### DISCUSSION TITLE
How to Run a Custom Method After Importing or Exporting List Items in My Admin View.

### DISCUSSION THREAD

**Discussion by philecom** [joomengine/Joomla-Component-Builder#768] [Q&A] [answered]
Hello,

I'm currently too way deep into JCB and I love it. I will first want to thank the developer and his immerse support on this platform.

My problem is, I want to run a custom method on the same Admin view table with the same selected items after IMPORTING or EXPORTING the list of records. 
What I want to achieve is 1) When I import a set of records into my admin view, I want to have the ids of all the newly imported records to perform further transactions on them. 2) When I export a list of records on the admin view list, I want to mark each record as exported after.
I've soo far not been able where to place the code.
I will be glad if anyone can point me to it. Thank you

**Comment by philecom** [comment]
any help?

**Comment by Llewellynvdm** [comment, marked as answer]
I would recommend that you watch [all tutorials](https://www.youtube.com/playlist?list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE)... believe me that is the best way.

The tutorial your looking for is this one: https://www.youtube.com/watch?v=fau5mZ6naLc&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=28

I know this is not in detail... but if I remember correctly it gets you in the door.
